airflow-core/src/airflow/api_fastapi/core_api/openapi/_private_ui.yaml

```diff
@@ -239,6 +239,48 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPValidationError'
+  /ui/dags/{dag_id}/latest_run:
+    get:
+      tags:
+      - DAG
+      summary: Get Latest Run Ui
```

Comment:
```suggestion
      summary: Get Latest Run
```

Reply:
this is autogen i believe, and based on the function name

Reply:
Ah, maybe we should rename it then?

Reply:
the openapi does not like it because there's another function with same name...

Reply:
done